ONSdigital/sdx-transform-cs
# QCAS Support

## What? and Why?
Added support for QCAS survey.

## How to test?
Test on sdx-console and ensure that the:
1. Data from eq submits successfully.
2. PCKs are valid. https://docs.google.com/spreadsheets/d/1DSLtru61W7F27K7tWZjsfMHJFN5kpcpS4qNsHy30LEY/edit#gid=108466985 
3. Images are as expected.

## Checklist
